frontend/src/components/v2/Select/components/index.tsx

```diff
@@ -0,0 +1,45 @@
+import {
```

Comment:
I think the folder structure might be a little confusing here. Inside the `Select` folder we already have an index.tsx which exports from `Select.tsx`. I would probably move this file into `components/v2/CreatableSelect/components/SelectComponents.tsx`, because these functions are purely related to the CreatableSelect component

Reply:
the select field child components are also used by `FilterableSelect` component, so I moved it to shared `Select` folder with the idea to later replace them all with the same `react-select` components
currently `Select` field is still using tailwind/radix dropdown as base